kafka_roller/kafka_roller/tasks.py

```diff
@@ -99,8 +103,16 @@ def stop(c, pre_stop=None):
     # introduce & execute post stop here if needed..
 
 
-def start(c):
+def start(c, pre_start=None):
     """Gracefully start an instance."""
-    # introduce & execute pre start here if needed..
+    if pre_start is not None:
+        result = run(c, pre_start)
+        if result.return_code != 0:
```

Comment:
nice, case for doing same in pre_stop?

Reply:
was thinking about it. seems like a good idea. how about a `--strict` mode?